### PR TITLE
Application GW url changes

### PR DESCRIPTION
**Pull request by kagamsft**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kagamsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kaga). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: .nuget/NuGet.Config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ogail) [inline .nuget/NuGet.Config]: Remove

File: .nuget/NuGet.targets [hunk 2]
```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ogail) [inline .nuget/NuGet.targets]: Remove

File: src/Authorization/Authorization.csproj [hunk 3]
```diff
@@ -41,4 +41,11 @@
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" Condition=" '$(LibraryFxTarget)' == 'portable' " />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition=" '$(LibraryFxTarget)' != 'portable' " />
   <Import Project="..\..\packages\Microsoft.Bcl.Build.1.0.14\tools\Microsoft.Bcl.Build.targets" Condition="Exists('..\..\packages\Microsoft.Bcl.Build.1.0.14\tools\Microsoft.Bcl.Build.targets')" />
+  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
```
Comment (ogail) [inline src/Authorization/Authorization.csproj]: Remove all changes not in ur project

**Comment by yugangw-msft** [comment]
please remove .nuget

**Pull request merged**